README.md

```diff
@@ -1,5 +1,15 @@
 # 02-OKJK
-Private repo for work by team OKJK
+
+Summary of Problem: There are a lot of economic indicators being captured by the Bureau of Labor Statistics (BLS). However, there is only a small set of the population that understands what these indicators mean. OKJK found common units between the data they were given by BLS. What they found was a relationship between the age, unemployment rates, and the price of an average grapefruit with the economic score of that state. As age, unemployment, and the price of a grapefruit increase, states’ economic scores decrease (this is reversed in the application due to an error in the model code). To demonstrate this, OKJK created an interactive web application that rates economic development based on user input relating to age, unemployment rate, and price of an average grapefruit. OKJK uses requests to get data from the Bureau. OKJK implements a pandas DataFrame which allows them to quickly combine, normalize, and fill in data. Their website runs on Heruko, a platform that allows developers to run applications entirely in the cloud. It uses Vincent for visualization and Flask for its framework. 
```

Comment:
one comment on this--its not so much that we've found relationships between these indicators. instead, what we've done is try to combine multiple economic indicators into a single score. the core idea behind the app is that it allows disparate economic indicators (like age, price of grapefruits, etc) be combined into one single economic score. the app is really about normalizing and displaying more than it is about analyzing any particular pattern.

also, the score reversal (unless i implemented it incorrectly [very possible]) is on purpose--since these indicators are 'bad things' we want states that have low values of them to have high scores. for example, alabama has a 5.8 local area unemployment rating and alaska has a 7.6. since its good that alabama has a lower unemployment rating, we want the score for alabama to be higher than alaska's, which the chart shows.